storage/store_metrics.go

```diff
@@ -74,9 +75,95 @@ var (
 	metaRangeSnapshotsPreemptiveApplied = metric.Metadata{Name: "range.snapshots.preemptive-applied"}
 
 	// Raft processing metrics.
-	metaRaftSelectDurationNanos  = metric.Metadata{Name: "process-raft.waitingnanos"}
-	metaRaftWorkingDurationNanos = metric.Metadata{Name: "process-raft.workingnanos"}
-	metaRaftTickingDurationNanos = metric.Metadata{Name: "process-raft.tickingnanos"}
+	metaRaftSelectDurationNanos = metric.Metadata{Name: "process-raft.waitingnanos",
+		Help: "Nanoseconds spent in store.processRaft() waiting"}
+	metaRaftWorkingDurationNanos = metric.Metadata{Name: "process-raft.workingnanos",
+		Help: "Nanoseconds spent in store.processRaft() working"}
+	metaRaftTickingDurationNanos = metric.Metadata{Name: "process-raft.tickingnanos",
+		Help: "Nanoseconds spent in store.processRaft() processing replica.Tick()"}
+
+	// Raft message metrics.
+	metaRaftRcvdMessages = map[raftpb.MessageType]metric.Metadata{
+		raftpb.MsgHup: {
+			Name: "raft-messages.rcvd.MsgHup",
```

Comment:
metric names get prometheus-ized to: `[a-zA-Z_:][a-zA-Z0-9_:]*` by replacing anything that doesn't match the regexp with `_`. This means that this name would get turned into: `raft_messages_rcvd_MsgHup`. Not great.

I'd also like to start normalizing the metrics names, at least for new ones. Existing ones will be renamed when refactors become easier (after the code yellow).

So for all of these, I propose:
`raft.received.hup`